_beats/libbeat/scripts/Makefile

```diff
@@ -450,4 +450,3 @@ release: mage
 
 .PHONY: snapshot
 snapshot: mage
-	@SNAPSHOT=true mage package
```

Comment:
leftover?

Reply:
yes! All changes in `_beats` and `vendor` will be replaced once the updated libbeat code is merged in.